packages/node-sdk/src/client.ts

```diff
@@ -546,41 +445,35 @@ export class BucketClientClass {
    * @remarks
    * Call `initialize` before calling this method to ensure the feature definitions are cached, no features will be returned otherwise.
    **/
-  public getFeatures(): Readonly<TypedFeatures> {
-    const mergedContext = {
-      user: this._user && {
-        id: this._user.userId,
-        ...this._user.attrs,
-      },
-      company: this._company && {
-        id: this._company.companyId,
-        ...this._company.attrs,
-      },
-      other: this._otherContext,
-    };
-
+  public getFeatures(context: Context): TypedFeatures {
```

Comment:
how does that work? You chave to do `boundClient.getFeatures(boundClient.context())`?

Reply:
I a missing some code I think as I do not see the connection between the `BoundClient` and the unbound one in terms of passing the context to these features.

Reply:
`BoundBucketClient` has a `context` and a `BucketClient` and when you call anything on the `BoundBucketClient` it'll just take it's context and call the corresponding method on the `BucketClient` passing along the context.

Reply:
I haven't noticed the:
```
  public getFeatures() {
    return this._client.getFeatures(this._context);
  }
```

before, hm